cookie_access_test.html

```diff
@@ -5,16 +5,20 @@
 </head>
 <body>
 <script>
-    // Try to set a cookie
-    var randVal = Math.random();
-    var randVal2 = 'test_' + Math.random();
-    document.cookie = randVal2+'='+randVal;
-
-    if (getCookie(randVal2) == randVal) {
-        window.parent.postMessage('cookies', "*");
-    } else {
-        window.parent.postMessage('no_cookies', "*");
-    }
+    // From: https://stackoverflow.com/a/3855394
```

Comment:
Can we use [`URLSearchParams`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams#Examples) instead?

Reply:
Done. Thanks for the suggestion.